### PR TITLE
Ewl 535 create player cards

### PR DESCRIPTION
**Pull request by ra-phael**
https://eurosportdigital.atlassian.net/browse/EWL-535

### CODE REVIEW

File: src/modules/PlayerCard/index.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  ul {
+    margin-bottom: 45px;
+  }
+`;
+
+const PlayerCard = ({ playerInfo, backgroundImageUrl, previousMatches }) => (
+  <StyledContainer>
+    <StyledBackground backgroundImageUrl={backgroundImageUrl} />
+    <StyledPlayerInfos player={playerInfo} />
+    <StyledPreviousMatches>PREVIOUS MATCHES</StyledPreviousMatches>
```
Comment (davidgarry) [inline src/modules/PlayerCard/index.js]: Display it only if there are some previous matches

File: src/modules/PlayerCard/index.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    <StyledMatches>
+      <ViewMore showLessText="View Less Matches" showMoreText="View More Matches">
```
Comment (davidgarry) [inline src/modules/PlayerCard/index.js]: The ViewMore button is broken on the storybook, it displays 4 items, and when I click on the button, it does not display the last item.

File: src/modules/PlayerCard/index.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+);
+
+const matchType = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  matchUrl: PropTypes.string.isRequired,
+  data: PropTypes.shape({
+    bottomTeam: teamDataType,
+    topTeam: teamDataType,
+  }).isRequired,
+  displayLeftCircle: PropTypes.oneOf(['won', 'lost', false]),
```
Comment (davidgarry) [inline src/modules/PlayerCard/index.js]: This property is not used as your code use `match.hasWon`
hasWon property is missing

File: src/modules/PlayerCard/index.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import * as colors from '../../colors';
+import { fontFamilies } from '../../typography';
+import PlayerInfos, { playerType } from '../../components/PlayerInfos';
+import ScoreBlock from '../../components/ScoreBlock';
+import { teamDataType } from '../../components/ScoreBlock/ScoreBlock';
```
Comment (davidgarry) [inline src/modules/PlayerCard/index.js]: It should be better to import it from `../../components/ScoreBlock`

File: src/modules/PlayerCard/index.js (hunk 2 above)
Comment (davidgarry) [inline src/modules/PlayerCard/index.js]: The View More button must be aligned with the player names and not centered 👍 
image

File: src/modules/PlayerCard/index.js (hunk 4 above)
Comment (ra-phael) [reply src/modules/PlayerCard/index.js]: Ok, I'll export teamDataType from ScoreBlock's index.js

File: src/modules/PlayerCard/index.js (hunk 2 above)
Comment (ra-phael) [reply src/modules/PlayerCard/index.js]: Thanks, I developed for mobile first and didn't notice this difference on desktop!

**Pull request merged**